app/routes/account_.$accountId.create/route.tsx

```diff
@@ -31,12 +32,11 @@ export const meta: MetaFunction = () => {
 //   price: Stripe.Price | void
 // }
 
-export const loader: LoaderFunction = async ({ request, params }) => {
+export const loader: LoaderFunction = async ({ request, params, context }) => {
```

Comment:
Is context being used?

Reply:
Nope. I added it to test something and forgot to remove it. Cleaned it up 👌